ScienceDirect.js

```diff
@@ -57,7 +57,7 @@ function detectWeb(doc, url) {
 
 function getPDFLink(doc, onDone) {
 	// No PDF access ("Get Full Text Elsewhere" or "Check for this article elsewhere")
-	if (doc.querySelector('.accessContent') || doc.querySelector('.access-options-link-text')) {
+	if (doc.querySelector('.accessContent') || doc.querySelector('.access-options-link-text') || doc.querySelector('#check-access-popover')) {
```

Comment:
Do we have an example where this is needed? Is it already part of the test cases?

Reply:
it's the standard PDF link behavior I now get when I'm not logged in.